xilem_web/web_examples/after_update/src/main.rs

```diff
@@ -1,26 +1,49 @@
 // Copyright 2023 the Xilem Authors
 // SPDX-License-Identifier: Apache-2.0
 
-use xilem_web::{document_body, elements::html, interfaces::Element, App};
+use xilem_web::{core::one_of::Either, document_body, elements::html, interfaces::Element, App};
 
-type AppState = bool;
+#[derive(Default)]
+struct AppState {
+    focus: bool,
```

Comment:
I think this should be either a `Rc<Cell<bool>>` or `Rc<AtomicBool>` (not sure which is more idiomatic in this context TBH), so that the focus value can be reset in the closure. to avoid calling `focus` on the element every rebuild.

You can see what I mean in action when adding somewhere something like this while typing in the input when `focus == true`:
```rust
            html::input(()).on_input(|_, _| {}),
```

I think somewhere in this example should probably also be comments, why this is an aweful hack (e.g. it doesn't trigger a rebuild after the callback was run), and just a necessary workaround to access the underlying DOM node.